src/tools/rbd/action/Qos.cc

```diff
@@ -0,0 +1,120 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
I really do not like the idea behind a one-off ```rbd``` CLI command to set a configuration setting. Right now we utilize the ```rbd image-meta``` commands to override configuration on a per-image basis. In the future, I can see that morphing into a collection of ```rbd config``` CLI commands for setting pool and image overrides.